functional/lenses.md

```diff
@@ -0,0 +1,481 @@
+# Lenses and Prisms
+
+This is a pure functional concept that is not frequently used in Rust, because
+other idioms can achieve the same result and more advanced forms are not
+supported.
+Never the less, exploring the concept may be helpful to understand other
```

Comment:
I've seldomly seen `Nevertheless`  spelled as three words. Is there are reason why you did spell it like this?